app/jobs/autoscaling.py

```diff
@@ -145,6 +147,16 @@ def autoscaling_update(config):
         else:
             print "No instances are ready to move from 'draining' to 'discarded' state."
 
+        if 'batch_configurations' in config:
+            print "Autoscaling batches:"
+            response_batch_autoscaling = batch_autoscaling.autoscale_compute_environments(config['batch_configurations'], dry_run=True)
```

Comment:
It would be nice if this dry_run option and similar could be configured based on tags attached to the Batch compute env.   That way we can turn this on or off without requiring a code push.   We already employ a similar technique for the rest of the autoscaling functionality, via a tag on the ASG called `IDSeqEnvsThatCanScale`.  The value of the tag is a list of environments whose autoscaler daemon has the right to modify that particular ASG.  You can entirely turn off the autoscaler (and turn it into a "dry run") by changing that tag to some value like "none".   Anyway, the point is that using tags we can make this config something that can be altered at runtime without a push.   It's valuable to have that ability when something isn't behaving quite right and we have lots of jobs running (when a push would be disruptive).

Reply:
I like this idea. Implementing it now.